### ISSUE TITLE
OSError: Cannot load native module 'Crypto.Cipher._raw_ecb' (Same error as #21 but v.3.2.0)

### ISSUE THREAD

**Issue by chorong761** [beardypig/streamlink-portable#32]
I'm experiencing the exact same error as Issue #21, but with version 3.2.0. I've tried py 3.6.8 x64, 3.7.9 x64 and 3.10.0 x64. All of them seem to have the same issue.

Tried it or two computers, one was overwriting an old version, one was the whole folder "clean install"

32-bit versions work tho

**Comment by wazerstar** [comment]
Yeah loading them into twitch gui will just say incorrect version..

 x86 works fine.
